configs/nr-otel-collector-agent-linux.yaml

```diff
@@ -132,32 +142,33 @@ processors:
       - key: type
         action: delete
 
+  transform:
+    trace_statements:
+      - context: span
+        statements:
+          - truncate_all(attributes, 4095)
+          - truncate_all(resource.attributes, 4095)
+    log_statements:
+      - context: log
+        statements:
+          - truncate_all(attributes, 4095)
+          - truncate_all(resource.attributes, 4095)
+
   # used to prevent out of memory situations on the collector
   memory_limiter:
     check_interval: 1s
     limit_mib: ${NEW_RELIC_MEMORY_LIMIT_MIB}
 
   batch:
 
-  cumulativetodelta:
-    include:
-      metrics:
-        - system.disk.operations
-        - system.disk.merged
-        - system.disk.io
-        - system.disk.io_time
-        - system.disk.operation_time
-        - system.network.io
-        - system.network.dropped
-        - system.network.packets
-      match_type: strict
   resource:
     attributes:
       - key: host.id
         from_attribute: host.name
         action: upsert
+
   resourcedetection:
-    detectors: ["env", "ec2", "system"]
+    detectors: ["env", "system"]
```

Comment:
is `host.name` populated when the `ec2` detector is removed? If not, we'd need to keep it and add azure and gcp too.